logs.txt

```diff
@@ -0,0 +1 @@
+nohup: failed to run command 'ypoetry': No such file or directory
```

Comment:
Looks like you had a typo when you tried to write the nohup command?

More importantly, this file should be removed (and added to .gitignore if it's being created by your Vagrant script).

Also you have downloaded + committed some `get-pip.py` files that should be deleted.